File: 2012/J3.py
x = int(raw_input())

count = 0
while count < x+1:
    print ('*' * x) + ('X' * x) + ('*' * x)
    count += 1

i = 0
while i < x+1:
    print (' ' * x) + ('X' * (x*2))
    i += 1
3
g = 0
while g < x+1:
    print ("*" * x) + (' ' * x) + ('*' * x)
    g += 1

File: 2019/J3.cpp
/*
 * Author: DynamicSquid
 */

#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main() {
    int num;
    cin >> num;

    vector<string> lines(num);
    for (int a = 0; a < num; ++a)
        cin >> lines[a];

    for (size_t a = 0; a < lines.size(); ++a) {
        char letter = lines[a][0];
        int count = 0;

        for (size_t b = 0; b < lines[a].length(); ++b) {
            if (lines[a][b] == letter) {
                count++;
            }
            else {
                cout << count << ' ' << letter << ' ';

                letter = lines[a][b];
                count = 1;
            }
        }

        cout << count << ' ' << letter << '\n';
    }
}
